tests/server/auth/test_sqlalchemy_store.py

```diff
@@ -29,29 +19,12 @@
 pytestmark = pytest.mark.notrackingurimock
 
 
-def _get_db_uri_from_env_var():
-    auth_config_path = os.environ.get(_AUTH_CONFIG_PATH_ENV_VAR)
-    if auth_config_path:
-        auth_config = read_auth_config(auth_config_path)
-        return auth_config.database_uri
-
-
 @pytest.fixture
 def store(tmp_sqlite_uri):
-    db_uri_from_env_var = _get_db_uri_from_env_var()
     store = SqlAlchemyStore()
-    store.init_db(db_uri_from_env_var if db_uri_from_env_var else tmp_sqlite_uri)
+    store.init_db(tmp_sqlite_uri)
```

Comment:
We should always use a fresh temporary database.

Reply:
should we also change `model_registry/test_sqlalchemy_store.py`? I think it's doing the same thing there

Reply:
The reason `model_registry/test_sqlalchemy_store.py` uses the same thing is because we run it against several databases such as PostgreSQL. For SQLite, we can just remove a file, but for PostgreSQL, we can't.

Reply:
I see, thanks!